test/e2e/crcsuite/crcsuite.go

```diff
@@ -62,6 +62,10 @@ func FeatureContext(s *godog.Suite) {
 		CheckCRCStatus)
 	s.Step(`^(stdout|stderr) (?:should contain|contains) "(.*)" if bundle (is|is not) embedded$`,
 		CommandReturnShouldContainIfBundleEmbeddedOrNot)
+	s.Step(`^execute (.*) command$`,
```

Comment:
This looks very generic because it can capture `execute oc version command` which we don't want. Can't we do this something like `^execute crc (.*) coomand$` which will make sure it only capture crc specific commands?